proxy/make.js

```diff
@@ -112,6 +112,30 @@ task('node', 'build the node binary', function() {
   console.log('Node prepared.');
 });
 
+task('pack-webext', 'pack the webextension for deployment', function() {
+  try {
+    execSync(`rm -f source.zip`);
+    execSync(`rm -f webext/webext.zip`);
+  } catch (error) {
+    console.log('Error removing zip files');
+  }
+  execSync(`git submodule update --remote`);
+  var version = process.argv[3];
+  console.log(version);
+  var manifest = require('./webext/manifest.json')
+  manifest.version = version;
+  writeFileSync('./webext/manifest.json', JSON.stringify(manifest, null, 2), 'utf8');
+  execSync(`git commit -am "bump version to ${version}"`);
+  try {
+    execSync(`git tag webext-${version}`);
+  } catch (error) {
+    console.log('Error creating git tag');
```

Comment:
Do we really want try/catch blocks? That is, would we want to upload a new version even though it doesn't have a git tag?

Reply:
This is partially why I don't have the script automatically push. I figured the script should display the error, and then the user can either fix the error manually or revert the git commit and re-run the script.

I could also have the catch block revert the git commit and return, but either way I think catching the error and responding to it is strictly better than just letting node fail.

Reply:
Alright, I added a new commit to undo all the changes this script does in the case that a `git tag` fails.